tests/e2e/unified_test.go

```diff
@@ -136,6 +137,10 @@ func testFixturesInSeries(ctx context.Context, t *testing.T, testPause bool, can
 	t.Run("fixtures", func(t *testing.T) {
 		fixtures := resourcefixture.Load(t)
 		for _, fixture := range fixtures {
+			if mustOnlyRunTests := os.Getenv("MUST_ONLY_RUN_TESTS"); mustOnlyRunTests != "" &&
```

Comment:
I think we are trying to turn away from regex based things and using the built-in `run` regex support  from `go test`. e.g.

```
E2E_KUBE_TARGET=envtest \
RUN_E2E=1 GOLDEN_REQUEST_CHECKS=1 E2E_GCP_TARGET=mock \
        go test -test.count=1 -timeout 60s -v ./tests/e2e -run TestPauseInSeries/fixtures/globalcomputeaddress 2>&1
```

Reply:
Ah, I didn't realize there is the build-in regex! I'll revert the change here.

Reply:
Done.